### PR TITLE
Add docs for ThemeSelectorResult

### PR DESCRIPTION
**Pull request by hishamco**
This issue here is to avoid the expectation of `ThemeName` many of the dev may expect the theme name is used in `ThemeSelectorResult` while the identifier is used instead. @agriffard if we need add remark on the docs please let me know where should we add it

### CODE REVIEW

**Comment by hishamco** [comment]
What's the wrong with https://nodejs.org/dist/v15.14.0?!!

File: src/OrchardCore/OrchardCore.DisplayManagement/Theming/ThemeSelectorResult.cs [hunk 1]
```diff
@@ -1,8 +1,19 @@
...
     {
+        /// <summary>
+        /// Gets or sets the priority for the selected theme.
+        /// </summary>
         public int Priority { get; set; }
+
+        /// <summary>
+        /// Gets or sets the theme name.
+        /// </summary>
+        /// <remarks>Avoid to use the theme name acciedently. <see cref="ThemeSelectorResult"/> is using the theme identifier instead.</remarks>
```
Comment (Skrypt) [inline src/OrchardCore/OrchardCore.DisplayManagement/Theming/ThemeSelectorResult.cs:16]: Avoid using the theme name as a technical value.

Else fix typo on "acciedently" if you keep the same sentence.

File: src/OrchardCore/OrchardCore.DisplayManagement/Theming/ThemeSelectorResult.cs [hunk 2]
```diff
@@ -13,7 +13,7 @@ public class ThemeSelectorResult
         /// <summary>
         /// Gets or sets the theme name.
         /// </summary>
-        /// <remarks>Avoid to use the theme name acciedently. <see cref="ThemeSelectorResult"/> is using the theme identifier instead.</remarks>
+        /// <remarks>Avoid to use the theme name accidentally. <see cref="ThemeSelectorResult"/> is using the theme identifier instead.</remarks>
```
Comment (deanmarcussen) [inline src/OrchardCore/OrchardCore.DisplayManagement/Theming/ThemeSelectorResult.cs:16]: Gramatically this sentence makes no sense
```suggestion
        /// <remarks>This is the theme identifier. By default this is also the theme name, but if a custom identifier has been specified, please use the identifier.</remarks>
```

I would actually put it in the summary, noone will ever see it in remarks.

Comment (hishamco) [reply src/OrchardCore/OrchardCore.DisplayManagement/Theming/ThemeSelectorResult.cs:16]: I'll see the experience in VS & propbably VS Code, how the remarks are show up or they aren't appear at all

Comment (hishamco) [reply src/OrchardCore/OrchardCore.DisplayManagement/Theming/ThemeSelectorResult.cs:16]: > This is the theme identifier. By default this is also the theme name, but if a custom identifier has been specified, please use the identifier.

The theme name is not working at all?!!

Comment (hishamco) [reply src/OrchardCore/OrchardCore.DisplayManagement/Theming/ThemeSelectorResult.cs:16]: We could summarize it with what Jasmin suggested "Avoid using the theme name as a technical value." plus mentioning to use theme identifier instead

Comment (ns8482e) [reply src/OrchardCore/OrchardCore.DisplayManagement/Theming/ThemeSelectorResult.cs:16]: This requires change when enabling theme from admin

as it uses theme name and not id

**Review by Piedone — APPROVED** [review]

**Pull request merged**